packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -257,6 +259,10 @@ export async function updateBuilders(
   output: Output,
   builderDir?: string
 ): Promise<string[]> {
+  // This update runs in the background so its best
+  // to wait just in case installBuilders() is slow.
+  await sleep(ms('30s'));
```

Comment:
This still isn't canceling the update logic if `stop()` is invoked before the 30s timeout. We want to prevent the update completely if the dev server is shut down before the timeout.

Reply:
i.e. with `clearTimeout()`

Reply:
Ah yes you're right. This will print `Waiting for builders update to complete` when you CTRL+C.

Do you suggest adding a new global variable for this timer and calling clearTimeout() in the stop() function?

Reply:
Ok done 👍